_rules/6cbcbe.md

```diff
@@ -0,0 +1,206 @@
+
+**Note:** Resolving the links includes potential redirects.
+
+## Assumptions
+
+* This rule assumes that the purpose of the links for links with identical names would not be ambiguous to users in general.
+
+## Accessibility support
+
+There are no major accessibility support issues known for this rule.
+
+## Background
+
+- [Understanding Success Criterion 2.4.9: Link Purpose (Link Only)](https://www.w3.org/WAI/WCAG21/Understanding/link-purpose-link-only.html)
+- [URL parsing](#https://www.w3.org/TR/html52/infrastructure.html#parsing-urls)
+
+## Test Cases
+
+### Passed
+
+#### Passed example 1
+
+Identical link text leads to identical URLs:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+```
+
+#### Passed example 2
+
+Links resolves to same page after redirect:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/redirect.html">Contact us</a>
+```
+
+#### Passed example 3
+
+Identical pages are located on different URLs:
+
+```html
+<a href="/test-assets/6cbcbe/index.html">Contact us</a>
+<a href="/test-assets/6cbcbe/index-copy.html">Contact us</a>
+```
+
+#### Passed example 4
+
+Same link text used for links going to pages where the content section is the same, but where the navigation options (bread crumbs and local sub menus) differ due to different placement in navigation hierarchy:
+
+```html
+<a href="/test-assets/6cbcbe/about/contact.html">Contact us</a>
+<a href="/test-assets/6cbcbe/careers/contact.html">Contact us</a>
+```
+
+#### Passed example 5
+
+URLs differ due to trailing slashes:
+
+```html
+<a href="/test-assets/6cbcbe/">Contact us</a> 
+<a href="/test-assets/6cbcbe">Contact us</a>
+```
+
+#### Passed example 6
+
+Pages contain different amounts of information and/or differently worded information, but fulfils same purpose in relation to the link:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Contact us</a>
+<a href="/test-assets/6cbcbe/page2.html">Contact us</a>
+```
+
+#### Passed example 7
+
+Pages have the same content but use different layouts:
+
+```html
+<a href="/test-assets/6cbcbe/page1.html">Contact us</a>
+<a href="/test-assets/6cbcbe/page3.html">Contact us</a>
+```
+
+#### Passed example 8
+
+Links created via scripting with explicit role of link, but leads to the same resource: 
+
+```html
+<span role="link"
+   onclick="location='/test-assets/6cbcbe/index.html'">
+Link text
+</span>
+
+<span role="link"
+   onclick="location='/test-assets/6cbcbe/index.html'">
+Link text
+</span>
+```
+
+### Failed
+
+#### Failed example 1
+
+Same link text used for links going to different resources:
+
+```html
+<a href="http://facebook.com">Follow us</a> 
+<a href="http://twitter.com">Follow us</a>
+```
+
+#### Failed example 2
+
+Same link text used for links going to web pages with same name, but with different information:
```

Comment:
Nothing in the applicability the name of resources.

Also be good to stay consistent with either "information" or "content" or "that do not serve an equivalent purpose"